python/cudf/cudf/pandas/_wrappers/pandas.py

```diff
@@ -371,7 +371,9 @@ def Index__setattr__(self, name, value):
     pd.SparseDtype,
     fast_to_slow=_Unusable(),
     slow_to_fast=_Unusable(),
-    additional_attributes={"__hash__": _FastSlowAttribute("__hash__")},
+    additional_attributes={
```

Comment:
Just confirming that the changes in this file are just formtting changes (no need to revert if so)

Reply:
Yes, it's just formatting.

Reply:
I looked through and I think its just formatting